defs/scenarios/pacemaker/node1_found.yaml

```diff
@@ -0,0 +1,13 @@
+checks:
+  node1-found:
+    requires:
+      property: core.plugins.pacemaker.PacemakerStatus.offline_nodes
+      op: contains
+      value: node1
+conclusions:
+  node1-found-needs-removal:
+    decision: node1-found
+    raises:
+      type: core.issues.PacemakerWarning
+      message: >-
+       A node with the hostname node1 is currently configured and enabled on pacemaker. This is related to bug: https://bugs.launchpad.net/charm-hacluster/+bug/1874719. You can remove the node by running the following command on the application-hacluster leader: juju run-action <application>-hacluster/leader delete-node-from-ring node=node1 --wait
```

Comment:
Can you wrap the line at char 79 like python code. Makes it much more readable (to be at least). The newlines wont appear in the output message.